common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -267,9 +271,6 @@ def create_response_element(self, **kwargs):
 
         *answer_attr*: The "answer" attribute on the tag itself (treated as an
         alias to "expect", though "expect" takes priority if both are given)
-
-        *group_label*: Text to represent group of inputs when there are
```

Comment:
Group label had been added in for the problem with multiple text inputs, but once question text was moved into the correct location, group label was no longer needed.